core/src/test/java/io/github/tcdl/config/MsbMessageOptionsTest.java

```diff
@@ -17,7 +17,7 @@ public void testGetWaitForResponsesConfigsNull() {
         MsbMessageOptions msbMessageOptions = new MsbMessageOptions();
         msbMessageOptions.setWaitForResponses(null);
 
-        assertEquals("expect 0 if MessageOptions.waitForResponses is null", 0, msbMessageOptions.getWaitForResponses());
+        assertEquals("expect 0 if MessageOptions.waitForResponses is null", Integer.valueOf(0), msbMessageOptions.getWaitForResponses());
```

Comment:
Does this change really improves readability? It introduces 17 chars instead of 1 with the same meaning.